services/basket/src/test/java/org/cryptimeleon/incentive/services/basket/ItemsTest.java

```diff
@@ -0,0 +1,90 @@
+package org.cryptimeleon.incentive.services.basket;
+
+import lombok.extern.slf4j.Slf4j;
+import org.cryptimeleon.incentive.services.basket.model.Item;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.cryptimeleon.incentive.services.basket.ClientHelper.getItems;
+
+@Slf4j
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class ItemsTest {
+    @Value("${basket-service.provider-secret}")
+    private String providerSecret;
+
+    private final Item firstTestItem = new Item("23578", "First test item", 235);
```

Comment:
I heard that this is a good way to organize tests, looks very clean to me :)

Reply:
Thanks